### PR TITLE
Simulated env for scale test.

### PR DESCRIPTION
**Pull request by soumasish**

### CODE REVIEW

File: scale-tests/configs/2018-10-23-spark-scale-simulation.env [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+NON_GPU_DISPATCHERS_OUTPUT_FILE="non-gpu-dispatchers.out"
+NON_GPU_DISPATCHERS_JSON_OUTPUT_FILE="${NON_GPU_DISPATCHERS_OUTPUT_FILE}-dispatchers.json" # NOTE: this name is built internally by the deploy-dispatchers.py script.
+NON_GPU_QUOTA_DRIVERS_CPUS=8
+NON_GPU_QUOTA_DRIVERS_MEM=20000
+NON_GPU_QUOTA_EXECUTORS_CPUS=8
+NON_GPU_QUOTA_EXECUTORS_MEM=20000
+
+# GPU dispatchers configuration ################################################
+
+GPU_NUM_DISPATCHERS=2
```
Comment (mpereira) [inline scale-tests/configs/2018-10-23-spark-scale-simulation.env]: This should be `0`.

File: scale-tests/configs/2018-10-23-spark-scale-simulation.env [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+KAFKA_CONFIG='scale-tests/configs/kafka-options.json'
+CASSANDRA_CLUSTER_COUNT=1
+CASSANDRA_CONFIG='scale-tests/configs/cassandra-options.json'
+
+# Non-GPU dispatchers configuration ############################################
+
+NON_GPU_NUM_DISPATCHERS=3
+NON_GPU_DISPATCHERS_OUTPUT_FILE="non-gpu-dispatchers.out"
+NON_GPU_DISPATCHERS_JSON_OUTPUT_FILE="${NON_GPU_DISPATCHERS_OUTPUT_FILE}-dispatchers.json" # NOTE: this name is built internally by the deploy-dispatchers.py script.
+NON_GPU_QUOTA_DRIVERS_CPUS=8
```
Comment (mpereira) [inline scale-tests/configs/2018-10-23-spark-scale-simulation.env]: This should be `4`.

File: scale-tests/configs/2018-10-23-spark-scale-simulation.env [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+NON_GPU_QUOTA_DRIVERS_MEM=20000
+NON_GPU_QUOTA_EXECUTORS_CPUS=8
```
Comment (mpereira) [inline scale-tests/configs/2018-10-23-spark-scale-simulation.env]: This should be `4`.

File: scale-tests/configs/2018-10-23-spark-scale-simulation.env [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+INFINITE_CONSUMER_BATCH_SIZE_SECONDS=10
+INFINITE_CONSUMER_SPARK_CORES_MAX=1
+INFINITE_CONSUMER_SPARK_EXECUTOR_CORES=1
+
+# Batch jobs configuration #####################################################
+
+NON_GPU_DISPATCHERS_JSON_OUTPUT_FILE_URL="https://${TEST_S3_BUCKET}.s3.amazonaws.com/${TEST_S3_FOLDER}/${NON_GPU_DISPATCHERS_JSON_OUTPUT_FILE}"
+
+BATCH_APP_ID="/${SERVICE_NAMES_PREFIX}batch-workload"
+BATCH_SCRIPT_CPUS=6
```
Comment (mpereira) [inline scale-tests/configs/2018-10-23-spark-scale-simulation.env]: This should be `4`.

File: scale-tests/configs/2018-10-23-spark-scale-simulation.env (hunk 3 above)
Comment (soumasish) [reply scale-tests/configs/2018-10-23-spark-scale-simulation.env]: Thanks @mpereira updated the configs.

**Pull request merged**